src/Slices/Settings/UI/Tabs/Configuration/Container.tsx

```diff
@@ -18,6 +18,20 @@ export const Container: React.FC<Props> = ({
   onErrorClose,
   className,
 }) => {
+  const [showUpdateBanner, setShowUpdateBanner] = useState(false);
+
+  useEffect(() => {
+    const updateSuccessBanner = () => {
+      setShowUpdateBanner(true);
+      setTimeout(() => {
+        setShowUpdateBanner(false);
+      }, 2000);
+    };
+    document.addEventListener("Settings update", updateSuccessBanner);
```

Comment:
I think we should have a standard for this or agree on a casing or something. Events in html are always lowercased. To keep it readable, we can dash them. so this one would be 
`settings-update`

If you agree of course, we could make a decision log for it in the notes (that's the place I've added decisions/logs about choices so far, until we have a dedicated place for that)

Reply:
Fine for me, I'll change and add info about this to notes